cdhweb/static_src/global/migrated-content.scss

```diff
@@ -2,7 +2,7 @@
     Regarding content migrated as part of Springload's 2024 rebuild.
     Existing content was dumped into a "migrated" content streamfield block.
 */
-.migrated-content {
+.block--migrated-content {
```

Comment:
This is another special case. I mean, I could add another wrapper but I don't intend to add any styling for this content at this stage.